x-pack/plugin/esql/src/internalClusterTest/java/org/elasticsearch/xpack/esql/action/TimeSeriesIT.java

```diff
@@ -586,15 +586,16 @@ public void testProfile() {
                 List<DriverProfile> dataProfiles = profile.drivers().stream().filter(d -> d.description().equals("data")).toList();
                 assertThat(dataProfiles, hasSize(1));
                 List<OperatorStatus> ops = dataProfiles.get(0).operators();
-                assertThat(ops, hasSize(5));
+                assertThat(ops, hasSize(6));
```

Comment:
This change adds an extra ValuesSourceReaderOperator step into this ESQL query - does that sound right to you @martijnvg ?

Reply:
Do you know for what field(s) both value source reader operations are?